### PR TITLE
fix broken mobile share in Files tab and uploading row style

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: shared/common-adapters/floating-menu/index.tsx [hunk 1]
```diff
@@ -38,6 +47,7 @@ export default (props: Props) => {
       attachTo={props.attachTo}
       style={props.containerStyle}
       propagateOutsideClicks={props.propagateOutsideClicks}
+      key={itemChangeCounter.toString()}
```
Comment (songgao) [inline shared/common-adapters/floating-menu/index.tsx]: No idea what's going on here, but without this forced re-render it just shows stale data.

Comment (chrisnojima) [reply shared/common-adapters/floating-menu/index.tsx]: Did you merge master for my fix and run modules.  Let's not do this

Comment (songgao) [reply shared/common-adapters/floating-menu/index.tsx]: I think so; this is pretty new-ish. OK I’ll try to figure out this weekend

**Comment by songgao** [comment]
Just tried the latest TestFlight which has @chrisnojima’s fix to react gateway and this is actually fixed. I just didn’t do yarn modules earlier.

**Pull request closed without merging**